packages/editor/src/components/autosave-monitor/index.js

```diff
@@ -91,6 +91,15 @@ export class AutosaveMonitor extends Component {
 	}
 }
 
+/**
+ * AutosaveMonitor component.
```

Comment:
We can skip this sentence because the description is generated after the name of the component. This way we don't have duplication of the name.